assets/rich_text_components_specs.js

```diff
@@ -0,0 +1,231 @@
+/* eslint-disable */
+/* Don't modify anything outside the {} brackets.
+ * Insides of the {} brackets should be formatted as a JSON object.
```

Comment:
The contents of the {} brackets should ...

Reply:
Done.